api/v1alpha1/jwt_types.go

```diff
@@ -8,6 +8,10 @@ package v1alpha1
 // JWT defines the configuration for JSON Web Token (JWT) authentication.
 type JWT struct {
 
+	// AllowMissing determines whether a missing JWT is acceptable, defaulting to false if not specified.
+	// Note: Even if allowMissing is set to true, JWT authentication will still fail if an invalid JWT is presented.
+	AllowMissing *bool `json:"allowMissing,omitempty"`
```

Comment:
hey @ardikabs, we discussed this in the community meeting, and most preferred `Optional` over `AllowMissing` , can we update this ?

Reply:
@arkodg 
can we keep the naming convention `AllowMissing` in the IR while using `Optional` in the user interface (CRD)?

Reply:
sure sgtm